### PR TITLE
Simplifying assertion on Path

### LINKED ISSUE #750: Simplify assertion on Path
Simplify assertion on Path.

```
    Truth.assertThat((Iterable<Path>) output)
        .containsAtLeastElementsIn(
            Paths.get("target","com.google.cloud", "libraries-bom", "index.html"))
        .inOrder();
```

### PR DESCRIPTION
**Pull request by suztomo**
Fixes #750 

- I found `containsAtLeastElementsIn` works better to handle Path, which is Iterable
- Fixing typo `snapshotlink`

### CODE REVIEW

File: dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java [hunk 1]
```diff
@@ -427,11 +427,8 @@ public void testOutputDirectory() {
     Truth.assertWithMessage(
             "The dashboard should be created at target/com.google.cloud/libraries-bom/1.0.0")
         .that((Iterable<Path>) outputDirectory)
-        .containsAtLeast(
-            Paths.get("target"),
-            Paths.get("com.google.cloud"),
-            Paths.get("libraries-bom"),
-            Paths.get("1.0.0"))
+        .containsAtLeastElementsIn(
```
Comment (elharo) [inline dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java]: do we care about order here? This test suggests we don't.

Comment (suztomo) [reply dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java]: Yes, we care about order. The `inOrder()` method 2 lines below verifies the order.

If order is wrong, then we get:
```
required elements were all found, but order was wrong
expected order for required elements: [com.google.cloud, libraries-bom, target, index.html]
but was                             : target/com.google.cloud/libraries-bom/index.html

	at com.google.cloud.tools.opensource.dashboard.FreemarkerTest.testVersionIndex(FreemarkerTest.java:129)
```

Comment (elharo) [reply dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java]: does this allow for intervening elements or not? API docs are unclear: https://github.com/google/truth/issues/594

Comment (suztomo) [reply dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java]: This allows intervening elements.

```
  @Test
  public void testIntervene() {
    // This passes
    Truth.assertThat(ImmutableList.of("a", "z", "b", "c"))
        .containsAtLeastElementsIn(ImmutableList.of("a", "b", "c")).inOrder();
  }

```

Comment (suztomo) [reply dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java]: Updated to containsExactlyElementsIn, which does not allow intervening elements.

**Comment by suztomo** [comment]
containsExactlyElementsIn only works for Linux.

```
[ERROR] Failures: 
[ERROR]   DashboardTest.testOutputDirectory:430 The dashboard should be created at target/com.google.cloud/libraries-bom/1.0.0
unexpected (7): Volumes, BuildData, tmpfs, src, github, cloud-opensource-java, dashboard
---
expected      : target/com.google.cloud/libraries-bom/1.0.0
but was       : /Volumes/BuildData/tmpfs/src/github/cloud-opensource-java/dashboard/target/com.google.cloud/libraries-bom/1.0.0
```

```
[ERROR] Failures: 
[ERROR]   DashboardTest.testOutputDirectory:430 The dashboard should be created at target/com.google.cloud/libraries-bom/1.0.0
unexpected (4): src, github, cloud-opensource-java, dashboard
---
expected      : target\com.google.cloud\libraries-bom\1.0.0
but was       : T:\src\github\cloud-opensource-java\dashboard\target\com.google.cloud\libraries-bom\1.0.0
```

File: dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java (hunk 1 above)
Comment (elharo) [reply dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java]: We could do that straight-forwardly. It's just Truth that doesn't support that.

**Pull request merged**